packages/vite/src/middleware/invokeMiddleware.ts

```diff
@@ -1,12 +1,14 @@
 import type { ServerAuthState } from '@redwoodjs/auth/dist/AuthProvider/ServerAuthProvider.js'
 import { setServerAuthState } from '@redwoodjs/server-store'
-
-import { MiddlewareRequest } from './MiddlewareRequest.js'
 import {
   MiddlewareResponse,
+  MiddlewareRequest,
   MiddlewareShortCircuit,
-} from './MiddlewareResponse.js'
-import type { Middleware, MiddlewareInvokeOptions } from './types.js'
+} from '@redwoodjs/web/dist/server/middleware.js'
+import type {
+  Middleware,
+  MiddlewareInvokeOptions,
+} from '@redwoodjs/web/dist/server/middleware.js'
```

Comment:
Same again about importing types from dist

Reply:
As before. Import from dist until exports are supported by this package.